### DISCUSSION TITLE
Want to know

### DISCUSSION THREAD

**Discussion by gouravsharma4t7** [catherineisonline/pizza-time-with-react#4] [General]
Hello 
I am new to ReactJS recently I started learning ReactJS can you tell me if this project is the latest or not I mean in this project you have used React with NextJS.

**Comment by catherineisonline** [comment]
Hi, yes I am using the latest version of React, you can check the package.json file for more detailed information on what I have used and the used version: https://github.com/catherineisonline/pizza-time-with-react/blob/main/package.json
No, I am not using Next.js.

**Reply by CyrusGerozaga2001** [reply]
Hello, where do we find the database?

**Comment by catherineisonline** [comment]
@CyrusGerozaga2001 
I have all the data in the file name data, here is the link: https://github.com/catherineisonline/pizza-time-with-react/tree/main/src/data

**Reply by CyrusGerozaga2001** [reply]
do i have to connect to any database? or none?

**Reply by catherineisonline** [reply]
For user registration, I am using the SQL serverless database of [PlanetScale](https://planetscale.com/). Yes, you need to register and connect your own database with the keys provided. Here is their [documentation](https://planetscale.com/docs/tutorials/connect-nodejs-app) for Node.js.

**Reply by CyrusGerozaga2001** [reply]
can i connect this to any database if i dont have planetscale?

**Reply by catherineisonline** [reply]
Yes, of course, just make sure to do it according to their documentation and adapt it to this project.

**Reply by kingalfarizi** [reply]
hey can u teach me how i can connect my database? im trying for two weeks now i cant figure it out :(

**Reply by kingalfarizi** [reply]
ive done following the documentation and yet i cant still use login/register, the loading is endless
